docs/UserGuide.md

```diff
@@ -57,12 +57,16 @@ Clientele+ seamlessly combines client contacts, payment tracking and more in one
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 
+* If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.
```

Comment:
By any chance does it give an error message? When I tried, it gave the error message: "Multiple values specified for the following single-valued field(s): <fields>"

Reply:
I will remove that part, as it does indeed give an error message when multiple values are specified for add/edit but not for whitelist